lib/formatting.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  rules: {
+    'arrow-parens': ['error', 'always'],
+    'array-bracket-spacing': ['error', 'never'],
+    'comma-dangle': ['error', 'never'],
+    'computed-property-spacing': ['error', 'never'],
+    'eol-last': 'error',
+    'no-mixed-spaces-and-tabs': 'error',
+    'no-spaced-func': 'error',
+    'no-trailing-spaces': 'error',
+    'object-curly-spacing': ['error', 'always'],
+    'quotes': ['error', 'single', 'avoid-escape'],
+    'semi': ['error', 'always'],
+    'space-infix-ops': 'error',
+    'spaced-comment': ['error', 'always'],
+    'keyword-spacing': ['error', { before: true, after: true }],
+    'template-curly-spacing': ['error', 'never'],
+    'semi-spacing': 'error'
+  }
+};
```

Comment:
I'm not seeing where these formatting rules make their way into the `node.js` and versioned rule sets?

Reply:
Oops. I added them directly to `base.js` because I learned about [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier), which a Prettier-permitting config could use to turn off all the formatting rules.